lib/floe/workflow/runner.rb

```diff
@@ -5,29 +5,42 @@ class Workflow
     class Runner
       include Logging
 
-      TYPES         = %w[docker podman kubernetes].freeze
       OUTPUT_MARKER = "__FLOE_OUTPUT__\n"
 
       def initialize(_options = {})
       end
 
+      @runners = {}
```

Comment:
Do we have to worry about threaded access to this?  I could see floe being used in a multi-threaded scenario in the future.